README.md

```diff
@@ -25,6 +25,34 @@ Package Name  | Version | Installation directory
 
 ## Usage
 
+### CI-beginner github example
+ 
+ 1. at your github repo click on the "Actions" button or go to https://github.com/[user]/[repo]/actions
+ 1. add a new workflow by clicking on "New workflow"
+ 1. click on  "set up a workflow yourself" to create you .yml configuration file
+ 1. paste this: (where `mySimpleTest1.m` is a test script of function placed in your repo)
+ ```
+name: moxunittest
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: joergbrech/moxunit-action@v1.1
```

Comment:
Could you just fix the major version here?

```
- uses: joergbrech/moxunit-action@v1
```

I think its about time for a 1.2 release and then the README would be outdated.

Reply:
ok, thanks for the fix! I am doings test in this repo https://github.com/jorgepz/Octave-Matlab-CI-testbed and it seems that both v1 and v1.1 work fine (at least for the minimal test I am running).
I'll fix my file.